tests/e2e/test_rosacli_node_pool.go

```diff
@@ -365,6 +365,74 @@ var _ = Describe("Edit nodepool",
 				Expect(len(common.ParseTuningConfigs(np.TuningConfigs))).To(Equal(0))
 			})
 
+		It("create nodepool with tuning config will validate well - [id:63179]",
+			labels.Medium,
+			func() {
+				tuningConfigService := rosaClient.TuningConfig
+				nodePoolName := common.GenerateRandomName("np-63179", 2)
+				tuningConfig1Name := common.GenerateRandomName("tuned01", 2)
+				tuningConfig2Name := common.GenerateRandomName("tuned02", 2)
+				tuningConfig3Name := common.GenerateRandomName("tuned03", 2)
+				nonExistingTuningConfigName := common.GenerateRandomName("fake_tuning_config", 2)
+
+				tuningConfigPayload := `
+		{
+			"profile": [
+			  {
+				"data": "[main]\nsummary=Custom OpenShift profile\ninclude=openshift-node\n\n[sysctl]\nvm.dirty_ratio=\"25\"\n",
+				"name": "%s-profile"
+			  }
+			],
+			"recommend": [
+			  {
+				"priority": 10,
+				"profile": "%s-profile"
+			  }
+			]
+		 }
+		`
+
+				By("Prepare tuning configs")
+				_, err := tuningConfigService.CreateTuningConfig(clusterID, tuningConfig1Name, fmt.Sprintf(tuningConfigPayload, tuningConfig1Name, tuningConfig1Name))
+				Expect(err).ToNot(HaveOccurred())
+				_, err = tuningConfigService.CreateTuningConfig(clusterID, tuningConfig2Name, fmt.Sprintf(tuningConfigPayload, tuningConfig2Name, tuningConfig2Name))
+				Expect(err).ToNot(HaveOccurred())
+				_, err = tuningConfigService.CreateTuningConfig(clusterID, tuningConfig3Name, fmt.Sprintf(tuningConfigPayload, tuningConfig3Name, tuningConfig3Name))
```

Comment:
I am not the feature reviewer. I have a confusion about why we need to prepare 3 tuning configures.

Reply:
As the related test case says so, If u tell I can remove the two because I thing it's of no use.